.github/workflows/main.yml

```diff
@@ -1,27 +1,54 @@
 name: Build specification
 on:
   push:
-    branches: [ main ]
```

Comment:
nit: just want to make sure this is intended.

Reply:
My linter seems to have auto-formatted this and a few other places. It has no real effect, but I am happy to split the bits that were refactored out into a seperate patch if you prefer?